client/mig-console/investigator.go

```diff
@@ -320,10 +320,10 @@ func investigatorCreator(cli client.Client) (err error) {
 	}
 	switch strings.ToLower(respv) {
 	case "yes":
-		fmt.Println("Investigator will have manifest management permissions\n")
+		fmt.Println("Investigator will have manifest management permissions")
```

Comment:
I'm not sure off hand but this one might have been intentional to break up the output into logical blocks during investigator creation